internal/pkgs/hwidmanager/controller.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 TII (SSRC) and the Ghaf contributors
+// SPDX-License-Identifier: Apache-2.0
+package hwidmanager
```

Comment:
Why not make this a simple additional RPC of the WifiManager?

Reply:
It could be, but I am under the impression that it may or may not be a WiFi MAC, so I thought I'd make the API more generic.

Reply:
Sure, makes sense e.g. if it would run in the host. Then let's separate this as explained below